### PR TITLE
fix(curriculum): update description for working with media

### LINKED ISSUE #56863: [ Full Stack Cert ] - update description for working with media lecture block
Let’s go with this more descriptive intro text

```md
        "intro": [
          "In these lecture videos, you will learn how to work with the <code>audio</code> and <code>video</code> elements as well as with SVG’s and more."
        ]
```

Here is the file you need to edit

https://github.com/freeCodeCamp/freeCodeCamp/blob/main/client/i18n/locales/english/intro.json#L1727-L1730

### PR DESCRIPTION
**Pull request by mithelan**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/how-to-open-a-pull-request/).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #56863

### CODE REVIEW

File: client/i18n/locales/english/intro.json [hunk 1]
```diff
@@ -1734,7 +1734,7 @@
       },
       "lecture-working-with-links": {
         "title": "Working with Links",
-        "intro": ["Learn about using links in HTML with these lecture videos."]
+        "intro": ["In these lecture videos, you will learn about the different <code>target</code> attribute values, absolute and relative links and the different links states."]
```
Comment (gikf) [inline client/i18n/locales/english/intro.json:1737]:
```suggestion
        "intro": [
          "In these lecture videos, you will learn about the different <code>target</code> attribute values, absolute and relative links and the different links states."
        ]
```
To make prettier happy.

File: client/i18n/locales/english/intro.json [hunk 2]
```diff
@@ -1726,7 +1726,7 @@
       },
       "lecture-working-with-media": {
         "title": "Working with Media",
-        "intro": ["Learn about using media in HTML with these lecture videos."]
+        "intro": ["In these lecture videos, you will learn how to work with the <code>audio</code> and <code>video</code> elements as well as with SVG’s and more."]
```
Comment (ilenia-magoni) [inline client/i18n/locales/english/intro.json:1729]:
```suggestion
        "intro": ["In these lecture videos, you will learn how to work with the <code>audio</code> and <code>video</code> elements as well as with SVG's and more."]
```

**Comment by gikf** [comment]
Hey @mithelan

Thank you for opening this pull request.

Related issue is limited to the "first time" code contributors to this repository. Because of that I'm going to close this PR. We would welcome future pull requests from you.

Thank you and happy coding.

**Comment by koSSmozoO** [comment]
****🫣

**Pull request closed without merging**